### PR TITLE
adding KOTS upgrade page, re-writing Customer Managed overview page, …

### PR DESCRIPTION
**Pull request by klehmer**
adding KOTS upgrade page, re-writing Customer Managed overview page rewording sentence on dependencies page

## Description & motivation
The primary motivation for this PR is to add documentation for how to upgrade KOTS for Customer Managed deployments as this needs to be done periodically to avoid issues with the Admin Console. This PR also includes a re-write of the Customer Managed overview page to make it more consistent with the other deployment overview pages as well as a small rewording of a sentence on the Customer Managed dependencies page.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [x ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

### CODE REVIEW

File: website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+```bash
+kubectl kots admin-console upgrade -n <namespace>
+```
+
+If you are upgrading your KOTS installation and have not tried the previous command, try this first before proceeding to the following section as manually upgrading may not be necessary. If the current installed version of KOTS is too old the upgrade command may fail and then the following instructions should be followed to manually upgrade the KOTS installation.
+
+## Manually upgrading the KOTS installation
+
+This section describes the steps needed to manually upgrade KOTS by deleting the current installation and re-installing the latest version. This process necesitates updating dbt Cloud to the latest stable version and as such is it is recommended that this process be executed along with a planned dbt Cloud upgrade. This process will not cause any downtime of the dbt Cloud application.
```
Comment (cmcarthur) [inline website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md]:
```suggestion
This section describes the steps needed to manually upgrade KOTS by deleting the current installation and re-installing the latest version. This process necessitates updating dbt Cloud to the latest stable version and as such is it is recommended that this process be executed along with a planned dbt Cloud upgrade. This process will not cause any downtime of the dbt Cloud application.
```

File: website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+Note: This process will erase the Version History in the Admin Console and force you to update to the latest stable version (meaning that you will lose the ability to roll back to prior versions). If you need to deploy a previous version of dbt Cloud please reach out to the Fishtown Analytics team.
```
Comment (cmcarthur) [inline website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md:32]: you should be more specific than `the Fishtown Analytics team` -- probably reach out to your account manager, or to support@getdbt.com

File: website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+- **Latest KOTS Plugin**: If you are deploying into an existing Kubernetes cluster, make sure the latest kots plugin is installed locally by running the following command.
+
+```bash
+curl https://kots.io/install | bash
+```
+
+If deploying into a VM, make sure the VM has the latest version installed by running the same command.
+
+- **Backup configuration**: This process will generate a backup of your configuration, but it is _always_ a good idea to create a manual backup of your dbt Cloud configuration (especially the encryption settings and database settings, which can cause dbt Cloud to break if lost or not entered correctly).
```
Comment (cmcarthur) [inline website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md:46]: this looks good for now, but it seems worth making a dedicated page explaining how to back up an onprem install at some point

File: website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+	--config-values "./dist/<dbt-cloud-v1?/>upstream/userdata/config.yaml"
+```
+
+Note that the config values path may need modified based on the current version of KOTS that is being upgraded from. 
+
+After the KOTS installer finishes, login to the Admin Console and verify that the config values are populated correctly in the console. Once verified click continue to proceed to the preflight checks. Once the preflight checks run successfully click continue again to deploy the latest stable version of dbt Cloud.
+
+#### Re-install kURL proxy (VM only)
+
+If running dbt Cloud in a VM (not recommended) the kURL installer will need to be run to re-install the kURL proxy to serve up the Admin Console.
```
Comment (cmcarthur) [inline website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md:97]: 😂

**Review by cmcarthur — APPROVED** [review]

**Review by drewbanin — APPROVED** [review]
LGTM

**Pull request merged**